patch/driver-pca954x-i2c-mux-deselect-on-exit-config-option.patch

```diff
@@ -0,0 +1,62 @@
+Adding PCA9548 I2C MUX deselect on exit config
+
+From: wadelnn <wadelnn@users.noreply.github.com>
+
+---
+ drivers/i2c/muxes/Kconfig           | 7 +++++++
+ drivers/i2c/muxes/i2c-mux-pca954x.c | 7 ++++++-
+ 2 files changed, 13 insertions(+), 1 deletion(-)
+
+diff --git a/drivers/i2c/muxes/Kconfig b/drivers/i2c/muxes/Kconfig
+index f6d313e..83a49d3 100644
+--- a/drivers/i2c/muxes/Kconfig
++++ b/drivers/i2c/muxes/Kconfig
+@@ -48,6 +48,13 @@ config I2C_MUX_PCA954x
+ 	  This driver can also be built as a module.  If so, the module
+ 	  will be called i2c-mux-pca954x.
+ 
++config I2C_MUX_PCA954X_DESELECT_ON_EXIT
```

Comment:
Because the PCA954x I2C MUX will cause get/set failed when i2c command is used frequently. This feature use deselect_on_exit flag to avoid i2c command failed.